app/views/finders/show.atom.builder

```diff
@@ -0,0 +1,21 @@
+atom_feed do |feed|
+  feed.title(@finder.name)
+
+  dates = []
+  @results.results.map(&:metadata).each do |r|
```

Comment:
I can't think of anything better, definitely worth moving into a Presenter regardless. Having some tests around it would be really good, and if we do find a better approach it's masked from the template anyway.